utilities/write_batch_with_index/write_batch_with_index.cc

```diff
@@ -426,11 +427,12 @@ Status WriteBatchWithIndex::PutLogData(const Slice& blob) {
 void WriteBatchWithIndex::Clear() { rep->Clear(); }
 
 Status WriteBatchWithIndex::GetFromBatch(ColumnFamilyHandle* column_family,
-                                         const DBOptions& options,
+                                         const DBOptions& /* options */,
```

Comment:
We cannot easily clean this up since this is a public API just like `DB::Get()`, right?

Reply:
Yes, exactly; we can only make such breaking changes in major versions (so 9.0 would be our next chance).